samples/qa-a11y/src/views/ReadAlert.js

```diff
@@ -18,7 +18,7 @@ const ReadAlertView = () => {
 	const onClick2 = onClick(false);
 
 	useEffect( () => {
-		if (window.PalmServiceBridge) {
+		if (window.WebOSServiceBridge ?? window.PalmServiceBridge) {
```

Comment:
Do we need this check? 
LS2Request has the check logic and this check logic in this sample looks weird as a reference sample for app developers.

Reply:
Right, `LS2Request` already has checking logic. But, this sample is designed as a sample app, so it looks not weird as an app developer.